src/hooks/useDisputes.js

```diff
@@ -55,10 +55,35 @@ export default function useDisputes() {
 }
 
 export function useDispute(disputeId) {
-  const [disputeProcessed, setDisputeProcessed] = useState(null)
   const courtConfig = useCourtConfig()
   const now = useNow() // TODO: use court clock
+
   const { dispute, fetching } = useSingleDisputeSubscription(disputeId)
+  const disputeProcessed = useProcessedDispute(dispute)
+
+  const disputePhase = getPhaseAndTransition(dispute, courtConfig, now)
+  const disputePhaseKey = disputePhase
+    ? convertToString(Object.values(disputePhase)[0])
+    : ''
+
+  return useMemo(() => {
+    if (fetching || (dispute && !disputeProcessed)) {
+      return { fetching: true }
+    }
+
+    return {
+      dispute: dispute
```

Comment:
should we use a function here to avoid returning with inline ternary operation?

Reply:
I don't think this is a ternary operation?

Reply:
`? ` `:` is the ternary operator

Reply:
Ah thought you meant nested operations. What's the problem to use it like this? Don't see a benefit of using a function for this simple operation.

Reply:
i always try not to compute inside the return statement, but if you prefer like this is up to you